server/src/sim.rs

```diff
@@ -156,6 +163,17 @@ impl Sim {
                 input,
                 self.cfg.step_interval.as_secs_f32(),
             );
+            if prev_node != position.node {
```

Comment:
Does the client need this data? Currently, I believe the client's version of `DualGraph` will always have an empty `entities` vec, although we already have something like this with the server's version of `DualGraph` always having `None` for the surface extraction.

Reply:
Clients already infer node -> entity mappings from position updates:

https://github.com/Ralith/hypermine/blob/45eaf09f93110751e46cfd9c26efe174c7adf7fb/client/src/sim.rs#L177-L180